### DISCUSSION TITLE
Trying to change the name of result folder

### DISCUSSION THREAD

**Discussion by Engg-blogger** [bpftrace/bpftrace#2986] [Development]
Hi Team - I was building bpftrace with nix build image. This command produces result folder, which can run the scripts. But i want to change the name of this folder as it will be shipped to customers. I was looking at flake file, but it doesn't mention anything. Could you please guide

**Comment by jordalgo** [comment]
@the-prachi Thanks for your question though I'm a bit confused. Can you explain a little more about what your goal is? I didn't quite understand the part about the folder being shipped to users.

**Reply by Engg-blogger** [reply]
So nix.md explains about running the command: nix build .#appimage
When i run this command, it produces an artifact under the name "result", which is an appimage. I can give permissions to this artifact and run my scripts with this. But I want to change the name of this artifact to something more meaningful. I am new to nix and flake and would like to understand how the name can be changed.

**Reply by danobi** [reply]
Use `cp`? Not sure why you need to change the `result` directory name

**Reply by Engg-blogger** [reply]
The appimage generated from CI workflow has the name bpftrace. Whereas, when i run nix build, it creates "result". I can do cp but wanted to get the correct name through nix configuration.

**Reply by jordalgo** [reply]
"result" is just a symlink to the image 
```
result -> /nix/store/44b0icx5f8n0hxiamgllpdw33j8b5gmd-bpftrace-aarch64.AppImage
```
but this name can be changed with the -o option:
https://nixos.org/manual/nix/stable/command-ref/nix-build#opt-out-link

**Reply by Engg-blogger** [reply]
Thanks. this is what I was looking for.
